powerline.el

```diff
@@ -112,6 +112,11 @@ This is needed to make sure that text is properly aligned."
   :group 'powerline
   :type 'boolean)
 
+(defcustom powerline-gui-hide-vcs-glyph t
```

Comment:
would you mind changing this to `powerline-gui-use-vcs-glyph` and just default to `nil`?

I think this should be an "enable" variable, rather than a "disable" variable.